### PR TITLE
US135204 [AQ] Go to Quiz link

### PR DESCRIPTION
**Pull request by Vitalii-Misechko**
[US135204](https://rally1.rallydev.com/#/?detail=/userstory/622335065533&fdp=true): [AQ] Go to Quiz link

* Functional Testing 
  * [x] Lit 2 Phase 1 compliant changes
  * [x] Browsers (Chrome, FF, Edge)
  * [x] Accessibility
  * [x] RTL
  * [x] Responsive
  * [x] Test cases
    * [x] `Go to quiz` link opens quiz in edit mode in a new browser tab
    * [x] `Go to quiz` link opens correct quiz when a user navigates to Quiz Item Analysis directly by link
* Security, devops, perf: N/A
* UX and docs

* [ ] Demo

  
FYI @Brightspace/draco-vis

### CODE REVIEW

File: pages/quiz-item-analysis-view.js [hunk 1]
```diff
@@ -433,10 +452,23 @@ class QuizItemAnalysisView extends SortMixin(SkeletonMixin(Localizer(MobxLitElem
 			'd2l-heading-1': true,
 			'd2l-skeletize': !this._quizName // show skeleton when quizName is unknown yet
 		};
+		const orgUnitId = this._quizDetail?.courseId;
```
[the remark this answers is not in the document]
Comment (Vitalii-Misechko) [reply pages/quiz-item-analysis-view.js:455]: Then the link is broken. But, orgUnitId is undefined only when a user navigates to Quiz Item Analysis directly in a browser. At that time the link is hidden behind a skeleton

**Review by bemailloux — APPROVED** [review]
LGTM! I love how the "go to another page" icon is mirrored for RTL

**Comment by Vitalii-Misechko** [comment]
> LGTM! I love how the "go to another page" icon is mirrored for RTL

Yeah. I didn't expect it works right away.

**Review by NicholasHallman — APPROVED** [review]
demo looked good too, LGTM!

**Comment by Vitalii-Misechko** [comment]
> demo looked good too, LGTM!

That was an accidental demo. It wasn't complete as well. So, I'm going to show on Monday how the link works

**Pull request merged**